pb/rpc.proto

```diff
@@ -49,6 +53,24 @@ message GetBlockHashResponse {
     bytes Hash = 1;
 }
 
+message GetValidatorAtIndexRequest {
+    uint32 Index = 1;
+}
+
+message GetValidatorAtIndexResponse {
```

Comment:
Let's make a new Validator message including all of the fields from:
https://github.com/phoreproject/synapse/blob/master/primitives/state.go#L10

Then the rpc can just return the validator message.